src/main/java/uk/gov/ons/census/action/messaging/ActionFulfilmentReceiver.java

```diff
@@ -50,16 +50,11 @@ public void receiveEvent(ResponseManagementEvent responseManagementEvent) {
       throw new RuntimeException(
           String.format("Cannot find case %s for fulfilment request.", caseId));
     }
-    String packCode =
+    String fulfilmentCode =
         responseManagementEvent.getPayload().getFulfilmentRequest().getFulfilmentCode();
-    Optional<Integer> questionnaireType = determineQuestionnaireType(packCode);
+    Optional<Integer> questionnaireType = determineQuestionnaireType(fulfilmentCode);
     if (questionnaireType.isEmpty()) {
-      log.with("caseId", caseId)
-          .with("PackCode", packCode)
-          .error("Unknown packCode in fulfilment request.");
-      throw new RuntimeException(
-          String.format(
-              "Unknown packCode %s in fulfilment request for caseId %s", packCode, caseId));
+      return;
```

Comment:
This will silently fail on any un mapped fulfilment code, I think it needs to explicitly ignore the UAC fulfilment codes so that it can still log an error and nack if we get something unexpected.

Reply:
Added some handling and a test added.